display/src/main/java/us/ilite/display/auton/AutonConfigDisplay.java

```diff
@@ -0,0 +1,298 @@
+package us.ilite.display.auton;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+// **2018 imports**
+// import us.ilite.frc.common.config.SystemSettings;
+// import us.ilite.frc.common.input.EDriverControlMode;
+// import us.ilite.frc.common.types.ECross;
+// import us.ilite.frc.common.types.ECubeAction;
+// import us.ilite.frc.common.types.EStartingPosition;
+// import us.ilite.frc.common.util.CSVLogger;
```

Comment:
You have done a very great job with commenting the old imports, however, it would be better if they were completely removed.